README.md

```diff
@@ -1,10 +1,89 @@
 # Python lib
 [![Build Status](https://travis-ci.org/neptune-ml/neptune-lib.svg?branch=master)](https://travis-ci.org/neptune-ml/neptune-lib)
 
-## Cleaning the directory
+# Getting started
 
-`make clean`
+## Experiment Api
 
-## Building a Python package
+### Get api-token
 
-`make build`
+```bash
+neptune account api-token get
+```
+
+### Establish session
+
+```python
+from neptunelib.credentials import Credentials
+credentials = Credentials(api_token=`YOUR_NEPTUNE_API_TOKEN`)
+```
+
+```python
+from neptunelib.session import Session
+session = Session(credentials=credentials)
+```
+
+### Instatiate a Project object
+
+```python
+project = session.get_projects('neptune-ml')['neptune-ml/Salt-Detection']
+```
+
+### Get leaderbaord dataframe
+
+```python
+leaderboard_df = project.get_leaderboard()
+```
+
+### Get experiment data
+
+```python
+experiments = project.get_experiments(id=['SAL-2342'])
+experiment = experiments[0]
+```
+
+1. get numeric channel values dataframe
+    
+```python
+    channel_df = experiment.get_numeric_channels_values('network_1 epoch_val iou loss')
+```
+    
+2. get hardware utilization dataframe 
+
+```python
+   channel_df = experiment.get_hardware_utilization()
+```
+
+
+# Installation
+
+## Get prerequisites
+* python versions `2.7/3.5/3.6` are supported
+* `neptune-cli` is useful but not a must. If you want to install it go
```

Comment:
Actually, it's the only way to get the API token.

Reply:
True, but in the (near) future we will have some neptune-agnostic things so I figured I'd set the tone already.